app/test/test_ftp.py

```diff
@@ -113,26 +113,6 @@ def test_server_check_positive(self):
         self.assertTrue(worker.check())
         server.terminate()
 
-    @unittest.skipUnless(os.getenv("CF_INSTANCE_GUID"), "CF-only test")
-    def test_cf_server_delete(self):
-        server = multiprocessing.Process(
-            target=serve,
-            args=(self.root,),
-            kwargs=self.params
-        )
-        server.start()
-        time.sleep(5)
-        worker = FTPWorker(**self.params)
-        with worker as active:
-            n = len(self.files)
-            for fn in active.filenames:
-                with self.subTest(fn=fn):
-                    self.assertTrue(active.delete(fn))
-                    n -= 1
-                    self.assertEqual(n, len(os.listdir(self.root)))
-
-        server.terminate()
-
     @unittest.skipIf(os.getenv("CF_INSTANCE_GUID"), "local-only test")
```

Comment:
This line can be deleted.  No need to test if you're in CF anymore as it'll never ever be true.